api/nodes/serializers.py

```diff
@@ -373,3 +381,37 @@ class Meta:
     @staticmethod
     def get_id(obj):
         return '{}:{}'.format(obj.node._id, obj.provider)
+
+class NodeInstitutionRelationshipSerializer(JSONAPIRelationshipSerializer):
+
+    id = ser.CharField(source='institution_id', required=False, allow_null=True)
+    type = TypeField(required=False, allow_null=True)
+
+    links = LinksField({
+        'self': 'get_self_link',
+        'related': 'get_related_link',
+    })
+
+    class Meta:
+        type_ = 'institution'
```

Comment:
My opinion is that this should be changed to 'institutions'.  I don't know if we should distinguish between single and plural.   When we create a node, type is "nodes" even though we're creating one node.